.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Format the codebase for the first time
+6ae7a064eb53980dd391f6bd6484c12efe7083b4
```

Comment:
```suggestion
b049841d5707d5bd87be516d8cda7be2a7585eae
```

Reply:
Ah dangit, totally forgot to update it after I rebased. Thanks! Fixed.